src/main/java/hudson/maven/PomInfo.java

```diff
@@ -112,7 +113,7 @@
      */    
     private final String artifactId;
 
-    public final Notifier mailNotifier;
+    public final @CheckForNull NotifierInfo mailNotifier;
```

Comment:
May break binary compatibility (detached code), but I do not see external usages

Reply:
The owning class is package-private so it should be safe.